doc/main/reference/try_put_and_wait.rst

```diff
@@ -0,0 +1,324 @@
+    #include <oneapi/tbb/flow_graph.h>
+
+Synopsis
+--------
+
+.. code:: cpp
+
+    namespace oneapi {
+        namespace tbb {
+            template <typename Output, typename Policy = /*default-policy*/>
+            class continue_node {
+            public:
+                bool try_put_and_wait(const continue_msg& input);
+            }; // class continue_node
+
+            template <typename Input, typename Output = continue_msg, typename Policy = /*default-policy*/>
+            class function_node {
+            public:
+                bool try_put_and_wait(const Input& input);
+            }; // class function_node
+
+            template <typename T>
+            class overwrite_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class overwrite_node
+
+            template <typename T>
+            class write_once_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class write_once_node
+
+            template <typename T>
+            class buffer_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class buffer_node
+
+            template <typename T>
+            class queue_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class queue_node
+
+            template <typename T, typename Compare = std::less<T>>
+            class priority_queue_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class priority_queue_node
+
+            template <typename T>
+            class sequencer_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class sequencer_node
+
+            template <typename T, typename DecrementType = continue_msg>
+            class limiter_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class limiter_node
+
+            template <typename T>
+            class broadcast_node {
+            public:
+                bool try_put_and_wait(const T& input);
+            }; // class broadcast_node
+
+            template <typename TupleType>
+            class split_node {
+            public:
+                bool try_put_and_wait(const TupleType& input);
+            }; // class split_node
+        } // namespace tbb
+    } // namespace oneapi
+
+Member functions
+----------------
+
+This section desc
+
+.. code:: cpp
+
+    template <typename Output, typename Policy>
+    bool continue_node<Output, Policy>::try_put_and_wait(const continue_msg& input)
+
+**Effects**: Increments the count of input signals received. If the incremented count is equal to the number
+of known predecessors, performs the ``body`` function object execution.
+
+Waits for the completion of the ``input`` in the Flow Graph meaning all tasks in each graph node related to ``input``
```

Comment:
Waits for the completion of the ``input`` in the Flow Graph, meaning all tasks associated with each node related to the ``input`` are executed, and no related objects remain in any buffer within the graph.

Reply:
Fixed